snowflake-table-grant-all/main.tf

```diff
@@ -0,0 +1,28 @@
+locals {
+  // https://docs.snowflake.com/en/user-guide/security-access-control-privileges.html#table-privileges
+  privileges = [
+    "SELECT",
+    "INSERT",
+    "UPDATE",
+    "TRUNCATE",
+    "DELETE",
+    "REFERENCES",
+  ]
+}
+
+resource snowflake_table_grant all {
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing version constraint for provider "snowflake" in "required_providers"